packages/dashboards/tests/unit/services/dashboard-data-test.js

```diff
@@ -222,7 +223,11 @@ module('Unit | Service | dashboard data', function(hooks) {
       }
     });
 
-    const makeFilter = ({ dimension }) => ({ dimension: { name: dimension } });
+    const makeFilter = ({ dimension }) => {
```

Comment:
Do we have any test asserts with this change?

Reply:
Yes the existing test checks that the correct filters are applied to the widget. Now the condition is that the filters have values so we would expect `dim4` to not be added while the others are added as usual